tools/install.sh

```diff
@@ -160,7 +160,7 @@ os_distro=$(hostnamectl |awk -F: '/Operating System:/{print $2}'|cut -f2 -d ' ')
 debian=("Debian" "Ubuntu")
 redhat=("Red" "CentOS" "Fedora" "Oracle") 
 
-if [[ " ${debian[*]} " =~ " $os_distro " ]]; then
+if [[ " ${debian[*]} " == *" $os_distro "* ]]; then
```

Comment:
These changes are far less readable and understandable. What are you trying to solve with them?

Reply:
because shellcheck (see #5160) fail without these change.

Reply:
@driusan The shellcheck tool is giving an error about the regex operator `=~` being used with quotation marks. The quotation marks mean that the RHS will match literally rather than as a regex, e.g. that if the string contains `^` this will be treated literally rather than as a metacharacter.

Instead of treating this expression as a regex, this change does a more basic substring check. The wildcards are necessary or it doesn't work.

Another solution is to unquote the variable but still use the regex operator, e.g.:

```bash
if [[ " ${debian[*]} " =~ $os_distro ]]; then
```

However since we're not actually using a regex here, the substring comparison is more appropriate.

Reply:
And for the record it seems to be a fairly common bash idiom from what I'm finding on Google, though I agree with you it's a bit hard to parse.